AdblockPlusSafari/NSString+MarkdownRenderer.h

```diff
@@ -0,0 +1,24 @@
+//
+//  NSString+MarkdownRenderer.h
+//  AdblockPlusSafari
+//
+//  Created by Pavel Zdeněk on 2.O.15.
+//  Copyright © 2015 Eyeo GmbH. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@class UIFont;
+
+@interface NSString (MarkdownRenderer)
+
+/**
+@param token a single character representing a span marker. Can be anything but Markdown
+ compatible is `*` for bold and `_` for italic
+@param font the font to substitute in the span
+@return self as attributed string with font changes applied to the found spans
+ */
+- (NSAttributedString*)markdownSpanMarkerChar:(NSString*)markerChar
```

Comment:
I'm fine with going with this approach for now (even though we normally use HTML markup, such as `<strong>` for this purpose so we can reuse translated strings across projects, but that is not that much of a problem).

But it isn't really a Markdown renderer, it's actually a renderer that can use arbitrary marker chars and use an arbitrary font for text marked by those. So I think it should have a different name. How about something like `TextRenderer`?